src/odemis/acq/stitching/_registrar.py

```diff
@@ -499,9 +501,9 @@ def getPositions(self):
         tile_positions = []
         dep_tile_positions = []
 
-        registered_positions = self._assemble_mosaic()
+        self.registered_positions = self._assemble_mosaic()
```

Comment:
Please update the docsting for `.getPositions()` to indicate that .registered_positions will be updated after the call.